packages/vaadin-combo-box/test/helpers.js

```diff
@@ -63,13 +57,34 @@ export const makeItems = (length) => {
   return Array(...new Array(length)).map((_, i) => `item ${i}`);
 };
 
+/**
+ * Returns first item of the combo box dropdown.
+ */
+export const getFirstItem = (comboBox) => {
+  return comboBox.$.dropdown._scroller.querySelector('vaadin-combo-box-item');
+};
+
+/**
+ * Returns selected item in the combo box dropdown.
+ */
+export const getSelectedItem = (comboBox) => {
+  return comboBox.$.dropdown._scroller.querySelector('[selected]');
+};
+
+/**
+ * Returns all the items of the combo box dropdown.
+ */
+export const getAllItems = (comboBox) => {
+  return Array.from(comboBox.$.dropdown._scroller.querySelectorAll('vaadin-combo-box-item'));
```

Comment:
I believe it's always desired that the item set obtained from `getAllItems` wouldn't contain items that are `hidden` (by iron-list logic), and preferably having the items in index order (not necessarily DOM order). I suggest adding:
```
.filter((item) => !item.hidden)
.sort((a, b) => a.index - b.index)
```
...which can then be removed from `getViewportItems`

Reply:
Done.